packages/inter-protocol/test/vaultFactory/vault-contract-wrapper.js

```diff
@@ -71,7 +71,7 @@ export async function start(zcf, privateArgs, baggage) {
     timer,
     quoteMint: makeIssuerKit('quote', AssetKind.SET).mint,
   };
-  const priceAuthority = makeFakePriceAuthority(options);
+  const priceAuthority = await makeFakePriceAuthority(options);
```

Comment:
This doesn't seem to end up in `terms`.  Its only use is in `E(priceAuthority)`.